transform_inline.ml

```diff
@@ -0,0 +1,312 @@
+open Instr
+open Edit
+
+(* Unlike `analysis_input`, this preserves the order of `formals`. This
+   facilitates matching the formals to the arguments passed at a callsite.*)
```

Comment:
I think we should probably change the type of analysis input. Obviously a set is not always optimal.